docs/tutorial/create_the_bundle.md

```diff
@@ -52,19 +52,24 @@ Target Directory [src/]:<enter>
 You must then specify which format the configuration will be generated as. Use yml, since it is what is used in eZ Platform itself. Of course, any other format could be used.
 
 ``` bash
-Configuration format (annotation, yml, xml, php, annotation): yml<enter>
+Configuration format (annotation, yml, xml, php) [xml]: yml<enter>
 ```
 
-Our bundle should now be generated. Navigate to `src/EzSystemsTweetFieldTypeBundle` and you should see the following structure:
+Our bundle should now be generated. Navigate to `src/EzSystems/TweetFieldTypeBundle` and you should see the following structure:
 
 ``` bash
-$ ls -l src/EzSystemsTweetFieldTypeBundle
-Controller
+$ ls -l src/EzSystems/TweetFieldTypeBundle
+Controller/
+DependencyInjection/
 EzSystemsTweetFieldTypeBundle.php
-Resources
+Resources/
+Tests/
 ```
 
 Feel free to delete the Controller folder, since you won’t use it in this tutorial. It could have been useful, had our Field Type required an interface of its own.
+Also, you can safely delete Resources/views/Default folder and Resources/config/routing.yml file, as they won't be needed. You should remove "ez_systems_tweet_field_type" entry in your app/config/routing.yml file as well.
```

Comment:
Also, you can safely delete the `Resources/views/Default` folder and `Resources/config/routing.yml`, as they won't be needed. You should remove the `ez_systems_tweet_field_type` entry from your app/config/routing.yml file as well.